scripts/cassandra-integration-test.sh

```diff
@@ -45,17 +47,19 @@ apply_schema() {
     --network host
   )
   docker build -t ${image} ${schema_dir}
+  # shellcheck disable=SC2068
   docker run ${params[@]} ${image}
 }
 
 run_integration_test() {
   local version=$1
   local schema_version=$2
-  local cid=$(setup_cassandra ${version})
+  local cid
+  cid=$(setup_cassandra "${version}")
   apply_schema "$2"
   STORAGE=cassandra make storage-integration-test
   exit_status=$?
-  trap "teardown_cassandra ${cid}" EXIT
+  trap "teardown_cassandra \${cid}" EXIT
```

Comment:
our intention here was to evaluate `$cid` right away, so that the trap command becomes `teardown_cassandra some_id`. Using `\$` prevents expansion altogether:
```shell
$ x=123
$ echo "teardown ${x}"
teardown 123
$ echo "teardown \${x}"
teardown ${x}
```

Reply:
could not find a workaround for this, so I disabled it

Reply:
Does this work?
```
teardown_cmd="teardown_cassandra ${cid}"
trap $teardown_cmd EXIT
```

Reply:
- No, it will produce this warning 
```bash
In scripts/cassandra-integration-test.sh line 61:
  trap "$teardown_cmd" EXIT
        ^-----------^ SC2064 (warning): Use single quotes, otherwise this expands now rather than when signalled.
```

- I think, disabling it, is the approach for it, if we want `$cid` to expand right away